doc/modules/mod_commands.md

```diff
@@ -0,0 +1,74 @@
+# MongooseIM's command set
+
+## Purpose
+
+This is a basic set of administration and client commands.
+Our goal is to provide a consistent, easy to use API for MongooseIM.
+Both backend and client commands provide enough information to allow auto-generating access methods.
+We currently use it in our admin and client REST API interface.
+In the future it may replace the current mongooseimctl implementation.
```

Comment:
"In the future it may..." -> "In the future, it may..."? (comma)

Reply:
'mongooseimctl' in single quotes?

Reply:
`backticks` would be even better.

Reply:
After consulting Joe, the comma doesn't go there.